internal/pkg/describe/static_site.go

```diff
@@ -97,7 +109,22 @@ func (d *StaticSiteDescriber) Describe() (HumanJSONStringer, error) {
 				URL:         uri.URI,
 			})
 		}
+		bucketName, err := d.s3Client.BucketName(d.app, env, d.svc)
+		if err != nil {
+			return nil, fmt.Errorf("get bucket name for %q env: %w", env, err)
+		}
+		tree, err := d.awsS3Client.GetBucketTree(bucketName)
+		if err != nil {
+			return nil, fmt.Errorf("get tree representation of bucket contents: %w", err)
+		}
+		if tree != "" {
```

Comment:
💭  I wonder if we should show 
```
Environment test
   <the tree>
Environment prod
  There is no file in this environment.
```
instead of 
```
Environment test
   <the tree>
```

Reply:
We have a manifest validation preventing users from deploying to an env if there are no files to upload. So if we message anything, it should be that the svc isn't deployed to the env, not that there are no files in the env. But maybe mentioning the env at all is kind of misleading.

Reply:
i see - that makes sense to me!

Reply:
Yeah, I think especially bc the env wouldn't be mentioned in the command output at all if it's not deployed to that env, we should just leave this as is. Let me know if you disagree!

Reply:
yeah I think I agree that we should leave it as is in this case 👀  I thought user could deploy a static site with 0 files, hence the proposal. Since this is not true, the current design is better!